Week2/homework/maartjes-work.js

```diff
@@ -45,17 +45,33 @@ const tuesday = [
 const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
+function computeTasksInHours(maartjesTasksX, celling) {
+  const marrtjeTasksInHours = maartjesTasksX
+    .map(task => task.duration / 60)
+    .filter(task => task >= celling);
+  return marrtjeTasksInHours;
+}
+
 function computeEarnings(tasks, hourlyRate) {
   // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  const Taskhours = computeTasksInHours(tasks, 2);
+  let rate = 0;
+  for (const task of Taskhours) {
+    const Total = task * hourlyRate;
+    rate += Total;
+  }
```

Comment:
Since we're concerned here with practicing array methods, why don't you try a `.reduce()` (preferred) or `.forEach()` here?